js/main.js

```diff
@@ -133,34 +133,70 @@ $(document).ready(function() {
 document.addEventListener("DOMContentLoaded", function(event) {
   var ingeniuscontainer = document.getElementById("main");
   var gghcontainer = document.getElementById("mainggh");
+  // gghcontainer.style.display = "none";
   mainSwitch.checked = false;
+  gghcontainer.classList.add("ggh-visuallyhidden");
   mainSwitch.addEventListener("click", function() {
     if (mainSwitch.checked) {
-      ingeniuscontainer.style.transform = "translateX(-100%)";
-      ingeniuscontainer.style.transition = "all 1s";
-      gghcontainer.style.transform = "translateX(0)";
-      gghcontainer.style.transition = "all 1s";
-      gghcontainer.classList.remove("fade-out");
-      ingeniuscontainer.classList.remove("fade-in");
-      gghcontainer.classList.add("fade-in");
-      ingeniuscontainer.classList.add("fade-out");
-      gghcontainer.style.transition = "all 1s";
-      gghcontainer.style.display = "block";
-      ingeniuscontainer.style.display = "block";
+      // ingeniuscontainer.style.transform = "translateX(-100%)";
+      // ingeniuscontainer.style.transition = "all 1s";
+      // gghcontainer.style.transform = "translateX(0)";
+      // gghcontainer.style.transition = "all 1s";
+      // gghcontainer.classList.remove("fade-out");
+      // ingeniuscontainer.classList.remove("fade-in");
+      // gghcontainer.classList.add("fade-in");
+      // ingeniuscontainer.classList.add("fade-out");
+      // gghcontainer.style.transition = "all 1s";
+      // gghcontainer.style.display = "block";
+      // ingeniuscontainer.style.display = "block";
```

Comment:
It's better to remove the lines than comment them out. You can always use git checkout to see code from past commits